src/api/VoteMonitor.Api.Form/Handlers/DeleteQuestionHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using VoteMonitor.Api.Form.Commands;
+using VoteMonitor.Entities;
+
+namespace VoteMonitor.Api.Form.Handlers
+{
+    public class DeleteQuestionHandler : IRequestHandler<DeleteQuestionCommand, bool>
+    {
+        private readonly VoteMonitorContext _context;
+        public DeleteQuestionHandler(VoteMonitorContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<bool> Handle(DeleteQuestionCommand request, CancellationToken cancellationToken)
```

Comment:
@onofreirazvan thanks for contributing. This handler should not allow deleting question that have notes or answers. can you please add this checks also. Thanks

Reply:
@idormenco  I've updated the handler to check for answers and notes